Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -575,6 +585,8 @@ def incl : N →ₗ⁅R,L⁆ M :=
   { Submodule.subtype (N : Submodule R M) with map_lie' := fun {_ _} ↦ rfl }
 #align lie_submodule.incl LieSubmodule.incl
 
+theorem injective_incl : Function.Injective N.incl := Subtype.coe_injective
```

Comment:
Do we bother with this lemma for other subtypes? I think we tend to avoid re-stating injectivity for bundled versions.

Reply:
OK dropped.